demos/continuous_batching/rag/client/rag_demo.ipynb

```diff
@@ -12,25 +12,104 @@
     "\n",
     "It assumes the model server is already deployed on the same machine on port 8000 with model `meta-llama/Meta-Llama-3-8B-Instruct` for `chat/completions` and `Alibaba-NLP/gte-large-en-v1.5` for `embeddings` and `BAAI/bge-reranker-large` for `rerank` endpoint.\n",
     "\n",
-    "Check https://github.com/openvinotoolkit/model_server/tree/main/demos/continuous_batching/rag/README.md to see how they can be deployed.\n",
+    "Check https://github.com/openvinotoolkit/model_server/tree/main/demos/continuous_batching/rag/cpp/README.md or \n",
+    "https://github.com/openvinotoolkit/model_server/tree/main/demos/continuous_batching/rag/python/README.md to see how they can be deployed.\n",
     "LLM model, embeddings and rerank can be on hosted on the same model server instance or separately as needed.\n",
     "openai_api_base , base_url parameters with the target url and model names in the commands might need to be adjusted. \n",
     "\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 11,
    "id": "2a498b22-fb7f-4fa1-a4d8-a0d983eb4565",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "[notice] A new release of pip is available: 24.2 -> 25.1.1\n",
+      "[notice] To update, run: c:\\program files\\python39\\python.exe -m pip install --upgrade pip\n"
+     ]
+    }
+   ],
    "source": [
     "%pip install -q -r requirements.txt"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 45,
+   "id": "7212515f-b59b-498c-a66a-f6c59de8fcab",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "9cd87aeb54ea41bb88eb2537c2ce5e13",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(RadioButtons(description='OVMS running with:', options=('CPP demo models', 'Python demo models'…"
```

Comment:
CPP demo is not a meaningful name to me. Maybe we could use 2 sets of used models?